packages/cli-lib/lang/en.lyaml

```diff
@@ -607,6 +607,21 @@ en:
             positionals:
               src:
                 describe: "Path to the local theme, relative to your current working directory."
+      module:
+        describe: "Commands for working with modules, including marketplace validation with the marketplace-validate subcommand."
+        subcommands:
+          marketplaceValidate:
+            describe: "Validate a module for the marketplace"
+            errors:
+              invalidPath: "The path \"{{ path }}\" is not a path to a folder"
+            logs:
+              validatingModule: "Validating module \"{{ path }}\" \n"
+            options:
+              json:
+                describe: "Output raw json data"
+            positionals:
+              src:
+                describe: "Path to the local module, relative to your current working directory."
```

Comment:
Is this `src` referring to the path on the local file system OR the path to the module in the Design Manager? It looks like we're making use of it in our [request to fetch the module's deps](https://github.com/HubSpot/hubspot-cli/pull/649/files#diff-98b83e9a5bde3137a78d32f0fd35d4aff65b825a813931c50ecd77a7dd050dcaR28), so I'm guessing that it's the src in the Design Manager. Should we update this copy to reflect that?

Reply:
Correct, forgot to fix these strings when changing paths to be DM-focused. Fixed 👍